docusaurus/docs/how-to-guides/data-source-plugins/create-data-frames.md

```diff
@@ -64,36 +61,7 @@ frame.name = 'http_requests_total';
 
 ## Read values from a data frame
 
-When you're building a panel plugin, the data frames returned by the data source are available from the `data` prop in your panel component.
-
-```ts
-function SimplePanel({ data: Props }) {
-  const frame = data.series[0];
-
-  // ...
-}
-```
-
-Before you start reading the data, think about what data you expect. For example, to visualize a time series you need at least one time field and one number field.
-
-```ts
-const timeField = frame.fields.find((field) => field.type === FieldType.time);
-const valueField = frame.fields.find((field) => field.type === FieldType.number);
-```
-
-Other types of visualizations might need multiple dimensions. For example, a bubble chart that uses three numeric fields: the X-axis, Y-axis, and one for the radius of each bubble. In this case, instead of hard coding the field names, we recommend that you let the user choose the field to use for each dimension.
-
-```ts
-const x = frame.fields.find((field) => field.name === xField);
-const y = frame.fields.find((field) => field.name === yField);
-const size = frame.fields.find((field) => field.name === sizeField);
-
-for (let i = 0; i < frame.length; i++) {
-  const row = [x?.values[i], y?.values[i], size?.values[i]];
-
-  // ...
-}
-```
+When you're building a panel plugin, the data frames returned by the data source are available from the `data` prop in your panel component. To learn how to use the data frames, refer to our documentation (TO DO: ADD LINK)
```

Comment:
This is not complete - and the whole page still seems in a mixed state of moving into two files (one for creating, one for reading and displaying)